crates/dbsp/src/trace/ord/vec/indexed_wset_batch.rs

```diff
@@ -532,10 +528,10 @@ where
         _dst_hint: Option<BatchLocation>,
     ) -> Self {
         Self {
-            lower1: batch1.layer.lower_bound(),
-            upper1: batch1.layer.lower_bound() + batch1.layer.keys(),
-            lower2: batch2.layer.lower_bound(),
-            upper2: batch2.layer.lower_bound() + batch2.layer.keys(),
+            lower1: 0,
```

Comment:
so these are still used?

Reply:
Still needed, the merger advances the lower pointers until they reach the upper pointers.

Reply:
yeah, these are used to track the position of a cursor.